packages/cloud-cognitive/src/components/StatusIcon/_status-icon.scss

```diff
@@ -112,6 +112,10 @@ $block-class: #{$pkg-prefix}-status-icon;
             animation: rotating 8s infinite linear;
```

Comment:
For `infinite` animations it might be worth adding a `prefers-reduced-motion` check. See [here](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)

```css
@media (prefers-reduced-motion) {
  .some-icon-with-animation {
    animation: rotating 8s infinite linear;
  }
}
```

Reply:
I added a media query that removes the animation if a user's settings are set to reduced motion:

```
...
 @each $themekey in $themes {
      .#{$block-class}--#{$theme}.#{$block-class}--#{$themekey}-#{$icon} {
        svg {
          @if $icon == in-progress {
            @media (prefers-reduced-motion) {
              animation: none;
            }
            // stylelint-disable-next-line carbon/motion-token-use, declaration-property-unit-whitelist
            animation: rotating 8s infinite linear;
            // stylelint-disable-next-line carbon/theme-token-use
            fill: clr($icon, $theme);
          } @else if $icon == running {
            transform: scaleY(-1);
...
```